### ISSUE TITLE
Package size

### ISSUE THREAD

**Issue by baweaver** [heimdalljs/heimdalljs-lib#84]
I was looking through a `node_modules` folder from a clean install of EmberJS and noticed that the `heimdalljs/dist/tests` folder was quite large while searching for `/tests?/` folders. Is this necessary to have in the dist folder, and if not can it be excluded from the build?

```bash
➜  ember_test git:(master) find ./node_modules/heimdalljs -type d -exec du -sh {} + | sort -r | head -n 20
...
772K    ./node_modules/heimdalljs/dist/tests
...
```

Thanks for your time in reading!

**Comment by twokul** [comment]
@baweaver thanks for reporting!

**Comment by baweaver** [comment]
```
➜  ember_test git:(master) ls -lah ./node_modules/heimdalljs/dist/tests
total 1544
drwxr-xr-x  5 _____  _____   170B Mar 22 21:49 .
drwxr-xr-x  7 _____  _____   238B Mar 22 21:49 ..
-rw-r--r--  1 _____  _____   253K Nov 15 15:09 bundle.cjs.js
-rw-r--r--  1 _____  _____   258K Nov 15 15:09 bundle.es.js
-rw-r--r--  1 _____  _____   256K Nov 15 15:09 bundle.umd.js
```

Just for reference on the size.

**Comment by twokul** [comment]
@baweaver good catch. we are actually bundling both tests and heimdall code as a part of our test output (seems like a bug I introduced while converting to typescript).

**Comment by hjdivad** [comment]
Thanks so much for reporting this @baweaver 

```
[I] ⋊> ~/tmp npm show --json heimdalljs@0.3.2 | jq .dist.tarball | xargs http get | tar tfz -                                                                                                                                                                                                                                                                                                                                                                                                     11:16:24
package/package.json
package/README.md
package/heimdall.js
package/dist/heimdalljs.cjs.js
package/dist/heimdalljs.es.js
package/dist/heimdalljs.iife.js
package/dist/heimdalljs.umd.js
package/dist/tests/bundle.cjs.js
package/dist/tests/bundle.es.js
package/dist/tests/bundle.umd.js
package/CHANGELOG.md

[I] ⋊> ~/tmp npm show --json heimdalljs@0.3.3 | jq .dist.tarball | xargs http get | tar tfz -                                                                                                                                                                                                                                                                                                                                                                                                     11:16:43
package/package.json
package/README.md
package/heimdall.js
package/dist/heimdalljs.cjs.js
package/dist/heimdalljs.es.js
package/dist/heimdalljs.iife.js
package/dist/heimdalljs.umd.js
package/CHANGELOG.md
```

**Comment by hjdivad** [comment]
@twokul nope this is a bug we've had for a bit

```
[I] ⋊> ~/tmp npm show --json heimdalljs@0.2.3 | jq .dist.tarball | xargs http get | tar tfz -                                                                                                                                                                                                                                                                                                                                                                                                     11:18:54
package/package.json
package/README.md
package/heimdall.js
package/dist/heimdalljs.cjs.js
package/dist/heimdalljs.es.js
package/dist/heimdalljs.iife.js
package/dist/heimdalljs.umd.js
package/dist/tests/bundle.cjs.js
package/dist/tests/bundle.es.js
package/dist/tests/bundle.umd.js
package/CHANGELOG.md
```

**Comment by hjdivad** [comment]
fixed in v0.2.4 and v0.3.3
